packages/ag-charts-community/src/chart/factory/seriesTypes.ts

```diff
@@ -17,8 +20,9 @@ const STACKABLE_SERIES_TYPES = new Set<SeriesOptionsTypes['type']>();
 const GROUPABLE_SERIES_TYPES = new Set<SeriesOptionsTypes['type']>();
 const STACKED_BY_DEFAULT_SERIES_TYPES = new Set<string>();
 const SWAP_DEFAULT_AXES_CONDITIONS: Record<string, (opts: unknown) => boolean> = {};
+const CUSTOM_DEFAULT_AXES_SWAPPERS: Record<string, SwapperFunction> = {};
 
-export function registerSeries(
+export function registerSeries( //<TOpts extends SeriesOptionsUnion>(
```

Comment:
Nit: Is the comment redundant?

Reply:
Oh... I thought that I had removed this 🙈 Well spotted! Thanks